doc/dev/release-procedure.md

```diff
@@ -0,0 +1,239 @@
+Got second flash read:
+80 81 82 83 84 85 86 87 88 89 8a 8b 8c 8d 8e 8f 90 91 92 93 94 95 96 97 98 99 9a 9b 9c 9d 9e 9f a0 a1 a2 a3 a4 a5 a6 a7 a8 a9 aa ab ac ad ae af b0 b1 b2 b3 b4 b5 b6 b7 b8 b9 ba bb bc bd be bf c0 c1 c2 c3 c4 c5 c6 c7 c8 c9 ca cb cc cd ce cf d0 d1 d2 d3 d4 d5 d6 d7 d8 d9 da db dc dd de df e0 e1 e2 e3 e4 e5 e6 e7 e8 e9 ea eb ec ed ee ef f0 f1 f2 f3 f4 f5 f6 f7 f8 f9 fa fb fc fd fe ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff 
+```
+
+#### Vivado - Known errors
+
+None at the moment.
+
+#### Vivado - Known critical warnings
+
+None at the moment.
+
+## Software repository
+
+Check out fresh software repository:
+
+```sh
+cd ../
+git clone https://github.com/lowRISC/sonata-software
+cd sonata-software
+git checkout $SONATA_SOFTWARE_RELEASE_SHA
+git submodule update --init --recursive
+```
+
+Build examples (use nix development environment in sonata-software to check it's working):
+```sh
+nix develop .
+xmake -P ./examples
+```
+
+### Simple demo
+Load simple demo on to board and make a copy for release:
+
+```sh
+# Copy for the release
+cp ./build/cheriot/cheriot/release/sonata_simple_demo.slot1.uf2  ../sonata_simple_demo_vX.Y.slot1.uf2
+# Program onto the FPGA
+cp ../sonata_simple_demo_vX.Y.slot1.uf2 /path/to/SONATA
+```
+
+The LCD should display a lowRISC logo and 'hello world', the user LEDs should display a walking pattern.
+
+Open screen:
+
+```sh
+screen /dev/ttyUSB2 921600
+```
+
+You should see output from the simple demo after you press the reset button (SW5):
+
+```
+bootloader: Loading software from flash...
+bootloader: Booting into program, hopefully.
+Led Walk Raw: Look pretty LEDs!
+```
+
+### Proximity test
+
+Load proximity sensor demo on to board
+
+```sh
+cp ./build/cheriot/cheriot/release/sonata_proximity_demo.slot1.uf2 /path/to/SONATA
+```
+
+Ensure you have an APDS-9960 prox/gesture/color sensor plugged into qwiic1.
+
+Check this has the same visual (LEDs and LCD) behaviour as the simple demo.
+The LCD should display a lowRISC logo and 'hello world', the user LEDs should display a walking pattern.
+Wave your hand over the proximity sensor, you should see the RGB LEDs fade up and down (one on the left is red, one on the right is green) as you move your hand.
+Red should fade up as your hand gets closer, green should fade down.
+Check that the RGB LED colours are as expected.
+This helps catch any issues that mix up the R,G,B values.
+
+Open picocom:
+
+```sh
+picocom /dev/ttyUSB2 -b 921600 --imap lfcrlf
```

Comment:
Why use `picocom` here but `screen` earlier?

Reply:
I've now put the alternative picocom command next to the first screen command. For the rest of the document just using screen.